style.css

```diff
@@ -655,3 +694,343 @@ label {
+    color: #fff;
+    column-gap: 30px;
+    justify-content: space-around;
+    margin-right: 70px;
+    display: flex;
+    text-align: center;
+    align-self: center;
+  }
+
+  .union {
+    display: none;
+  }
+
+  .icon {
+    display: flex;
+    flex-direction: column;
+    row-gap: 15px;
+  }
+
+  .recent-work {
+    width: 1440px;
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .recent-header {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+  }
+
+  .Works {
+    padding: 0;
+    grid-template-columns: 350px 350px 369px;
+    grid-template-rows: 500px 410px 480px;
+    align-items: center;
+  }
+
+  .card {
+    padding: 0;
+    margin: 0;
+    width: 369px;
+    height: 435px;
+  }
+
+  .work-img {
+    padding: 0;
+    margin: 0;
+    grid-column: 1/3;
+    width: 684px;
+    height: 448px;
+    flex: none;
+  }
+
+  .work2 {
+    padding: 0;
+    margin: 0;
+    margin-top: -20%;
+    grid-column-start: 3;
+  }
+
+  .wtext1 {
+    padding: 0;
+    margin: 0;
+    grid-column: 2/3;
+  }
+
+  .work1 {
+    display: flex;
+  }
+
+  .line {
+    margin: 0;
+    width: 794px;
+  }
+
+  .works_header-text {
+    align-self: center;
+    display: flex;
+  }
+
+  .link-button2 {
+    display: flex;
+    align-items: center;
+    margin-bottom: 10%;
+    width: 327px;
+    height: 48px;
+    background: #ff6b00;
+    border: none;
+  }
+
+  .li {
+    margin: 0;
+  }
+
+  .About-myself {
+    padding: 0;
+    margin: 0;
+    background: #262626;
+    background-image: url("/img/about.png");
+    background-repeat: no-repeat;
+    width: 1440px;
+    height: 1195px;
+    flex-direction: column;
+    display: flex;
+    align-items: flex-start;
+  }
+
+  .about-text-part {
+    row-gap: 20px;
+    margin-left: 2%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .regular-text2 {
+    width: 450px;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    height: 148px;
+  }
+
+  .about-text {
+    padding: 0;
+    margin: 0;
+    margin-top: 15%;
+    display: flex;
+    align-self: flex-start;
+  }
+
+  .resume {
+    margin: 0;
+  }
+
+  .line2 {
+    width: 1156px;
+    margin: 0;
+    margin-top: 10%;
+    margin-left: 5%;
+    margin-bottom: 10%;
+    align-self: flex-start;
+  }
+
+  ul.menu {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: row;
+    column-gap: 35%;
+  }
+
+  .lang {
+    align-content: center;
+    align-items: flex-start;
+  }
+
+  ul.lang {
+    align-content: center;
+    margin-left: 25%;
+  }
+
+  .ico {
+    align-items: none;
+  }
+
+  .fo {
+    margin-left: -25%;
+  }
+
+  .contact {
+    align-self: center;
+    padding: 0;
+    margin: 0;
+    display: grid;
+    grid-template-columns: 400px 648px 100px 100px;
+    grid-template-rows: 550px 10px 50px;
+    align-items: start;
+    justify-content: center;
+    margin-top: 10%;
+    width: 1440px;
+  }
```

Comment:
- [ ] Dear @mohisa302, As I have mentioned in the other comments the fixed `1440px` display is causing your webpage to be less responsive to different screen sizes. Please use something like `100%` instead and work with that. I hope this will make a difference. 
- [ ] Also, As mentioned in the previous comments, Always better to use a relative `./` to refer to your image and resources. This will tell the browser to look inside the current repository.
- [ ] One more thing is the design, try your best to get the same design as the Figma design. This includes, alignment, spacing, color, and decorations. 

### Your design 
![Screenshot 2022-12-08 193225](https://user-images.githubusercontent.com/69077061/206510555-89cb910e-c7f2-4025-9a63-bc5568667a62.png)

### Figma design 
![Screenshot 2022-12-08 193216](https://user-images.githubusercontent.com/69077061/206510547-b404e4c7-7ade-4895-b0ce-83b4f2d93a5c.png)